src/Package/Test/FakeFactories/TestRInteractiveWorkflowProviderFactory.cs

```diff
@@ -44,7 +45,7 @@ public static class TestRInteractiveWorkflowProviderFactory {
             shell = shell ?? VsAppShell.Current;
             settings = settings ?? RToolsSettings.Current;
 
-           return new TestRInteractiveWorkflowProvider(sessionProvider, connectionsProvider, historyProvider, packagesProvider, plotsProvider, activeTextViewTracker, debuggerModeTracker, brokerConnector, shell, settings);
+            return new TestRInteractiveWorkflowProvider(sessionProvider, connectionsProvider, historyProvider, packagesProvider, plotsProvider, activeTextViewTracker, debuggerModeTracker, brokerConnector, shell, settings) { TestName = testName };
```

Comment:
Sure

---
In reply to: [75031291](https://github.com/Microsoft/RTVS/pull/2173#discussion_r75031291) [](ancestors = 75031291,75012197,74973702)